GettingStartedDeb.md

```diff
@@ -0,0 +1,111 @@
+> Tested on: Ubuntu 14.04, Mint 17.01
+
+As with all other operating systems you need DNVM to get going with ASP.NET 5. To get it you run `curl` to download a `.sh` file and then run it. To configure a Linux machine to run an ASP.NET 5 application use the following instructions.
+
+The steps to set up a Linux machine are:
+
+ * Get a working version of Mono
+ * Get and compile libuv (Required for the Kestrel server)
+ * Get DNVM
+ * Add sources to NuGet.config (For package restore)
+
+### Docker
+
+Instructions on how to use the ASP.NET [Docker](https://www.docker.com/) image here: http://blogs.msdn.com/b/webdev/archive/2015/01/14/running-asp-net-5-applications-in-linux-containers-with-docker.aspx
+
+The rest of this section deals with setting up a machine to run applications without the Docker image.
+
+### Get Mono
+
+Mono is how .NET applications can run on platforms other than Windows. Mono is an ongoing effort to port the .NET Framework to other platforms. In the process of developing ASP.NET 5 we worked with the Mono team to fix some bugs and add features that are needed to run ASP.NET applications. Because these changes haven't yet made it into an official Mono release we will either grab a Mono nightly build or compile Mono from source.
+
+#### Option 1: Mono CI build
+
+The Mono CI server builds packages for Linux distributions on each commit. To get them you install a particular snapshot and then run `mono-snapshot APP/VER` to change the current shell to use the provided snapshot. In these instructions we will grab the latest snapshot and set it to be the one to use.
+
+**NOTE: Mono snapshots do not persist outside the current shell, you need to run `mono-snapshot` each time you want to run the newer version of Mono. If this isn't what you want then look at compiling Mono from source option, the instructions here show building from source and installing Mono. If you want other options then you should follow the links to the Mono build instructions. **
+
+To do this we need to add the Mono CI server to apt-get:
+
+```bash
+sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
+echo "deb http://jenkins.mono-project.com/repo/debian sid main" | sudo tee /etc/apt/sources.list.d/mono-jenkins.list
+sudo apt-get update
+sudo apt-get install mono-snapshot-latest
+. mono-snapshot mono
+```
+**NOTE:** Official Mono instructions that these steps come from are here: http://www.mono-project.com/docs/getting-started/install/linux/ci-packages/.
+
+#### Option 2: Build Mono from source
+
+Building Mono from source can take some time, and the commands below will install the built version of Mono on your machine replacing any version you might already have.
+
+```bash
+sudo apt-get install git autoconf libtool automake build-essential mono-devel gettext
+PREFIX='/usr/local'
+PATH=$PREFIX/bin:$PATH
+git clone https://github.com/mono/mono.git
+cd mono
+./autogen.sh --prefix=$PREFIX
+make

```

Comment:
you might want to add `-j4` as this greatly speeds up the build (just assuming here that most developers have at least a quad-core machine :smile: )